python/lsst/meas/base/plugins.py

```diff
@@ -198,8 +198,12 @@ def measure(self, measRecord, exposure):
         # Compute the median variance value for each pixel not excluded by the mask and write the record.
         # Numpy median is used here instead of afw.math makeStatistics because of an issue with data types
         # being passed into the C++ layer (DM-2379).
-        medVar = numpy.median(pixels.getVarianceArray()[logicalMask])
-        measRecord.set(self.varValue, medVar)
+        if numpy.sum(logicalMask) > 0:
```

Comment:
Why not `numpy.any(logicalMask)`? That seems more explicit, and is (presumably) faster than summing for large masks.